src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBoxTests.cs

```diff
@@ -511,6 +511,109 @@ public void ComboBox_DropDownStyle_Set_GetReturnsExpected(ComboBoxStyle value)
         Assert.False(control.IsHandleCreated);
     }
 
+    [WinFormsTheory]
+    [InlineData(DrawMode.Normal)]
+    [InlineData(DrawMode.OwnerDrawFixed)]
+    [InlineData(DrawMode.OwnerDrawVariable)]
+    public void ComboBox_GetItemHeight_Invoke_ReturnsExpected(DrawMode drawMode)
+    {
+        const int index = 0;
+        const int expected = 15;
+        using ComboBox control = new()
+        {
+            DrawMode = drawMode,
+            ItemHeight = expected
+        };
+        control.Items.Add("Item1");
+        control.Items.Add("Item2");
+        control.Items.Add("Item3");
+
+        control.GetItemHeight(index).Should().Be(expected);
+    }
+
+    [WinFormsTheory]
+    [InlineData(-1)]
+    [InlineData(0)]
+    [InlineData(2)]
+    [InlineData(3)]
+    [InlineData(4)]
+    public void ComboBox_GetItemHeight_Index_ThrowsArgumentOutOfRangeException(int index)
+    {
+        using ComboBox control = new();
+        control.DrawMode = DrawMode.OwnerDrawVariable; // Set the DrawMode property
+        control.CreateControl(); // Make sure the IsHandleCreated property is true
+        control.Items.Add("Item1");
+        control.Items.Add("Item2");
+        control.Items.Add("Item3");
+
+        if (index < 0 || index >= control.Items.Count)
+        {
+            control.Invoking(y => y.GetItemHeight(index))
+                   .Should().Throw<ArgumentException>()
+                   .Where(ex => ex.ParamName == "index");
+        }
+        else
+        {
+            // Test valid index
+            int itemHeight = control.GetItemHeight(index);
+            itemHeight.Should().BeGreaterThan(0, "Item height should be greater than 0.");
+        }
+    }
+
+    [WinFormsFact]
+    public void ComboBox_GetItemHeight_NoItems_ThrowsArgumentOutOfRangeException()
+    {
+        using ComboBox control = new();
+        control.DrawMode = DrawMode.OwnerDrawVariable;
+        control.CreateControl();
+        control.Invoking(c => c.GetItemHeight(0))
+               .Should().Throw<ArgumentOutOfRangeException>();
+    }
+
+    [WinFormsTheory]
+    [MemberData(nameof(BoolData))]
```

Comment:
There is already an existing bool data that can be used
```suggestion
    [BoolData]
```